### PR TITLE
Add and configure Cow changelog template

### PR DESCRIPTION
**Pull request by Cheddam**
Cow can now use a Twig template as a base for generating changelogs. This PR introduces one based on the format of changelogs from recent minor SilverStripe releases.

### CODE REVIEW

**Comment by dnsl48** [comment]
maybe that's time to create a folder `.cow` and move `.cow.json -> .cow/config.json`, `changelog.md.twig -> .cow/changelog.md.twig`?

**Comment by ScopeyNZ** [comment]
I think in order to not require a change to cow it's an acceptable solution is to move this twig template into a `.cow` folder and update the config. Changing `.cow.json` requires a cow update.

File: .cow.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "github-slug": "silverstripe/silverstripe-installer",
   "changelog-holder": "silverstripe/framework",
   "changelog-path": "docs/en/04_Changelogs/{stability}/{version}.md",
+  "changelog-template": ".changelog.md.twig",
```
Comment (dnsl48) [inline .cow.json]: Sorry, I'm not sure what's happening :) Shouldn't this become `.cow/.changelog.md.twig`?

Comment (Cheddam) [reply .cow.json]:
```suggestion
  "changelog-template": ".cow/.changelog.md.twig",
```

**Pull request merged**